recipe/meta.yaml

```diff
@@ -45,7 +45,7 @@ test:
     - mocpy_test.py
   imports:
     - mocpy
-    - astropy
+    - astropy-base
```

Comment:
This is a bug in the migration bot, bot I'm not sure it worth fixing centrally. cc @cadair @beckermr
```suggestion
    - astropy
```

Reply:
(I mean, I don't think astropy should be even listed in here, so one could argue this is a recipe bug)

Reply:
This line should not be in the recipe. Remove it and move on.